### ISSUE TITLE
heterochromatin domain assembly, vs heterochromatin assembly

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#11247]
I hadn't seen 
heterochromatin domain assembly before

The aggregation, arrangement and bonding together of a set of components to form a heterochromatin domain. 

how does this differ from the parent heterochromatin assembly (i.e isn't  couldn't all heterochromatin be referred to as a heterochromatin domain?)

Reported by: ValWood

Original Ticket: [geneontology/ontology-requests/11065](https://sourceforge.net/p/geneontology/ontology-requests/11065)

**Comment by paolaroncaglia** [comment]
Sitting with Val now, she still needs time to look into this, and it's still low priority.

**Comment by paolaroncaglia** [comment]
Hi @ValWood,
I’m going to unassign myself from this ticket. However, rather than leaving it unassigned, I’ll give it to you. If it’s no longer relevant, please close it. Otherwise, perhaps when you have had time to look into it, you could collate the relevant info in a new ticket - and close this one. Thanks.

**Comment by ValWood** [comment]
No problem,
val

**Comment by pgaudet** [comment]
Merge ?

**Comment by ValWood** [comment]
I'm not sure.

Heterochromatin island is defined

Definition |
A region of heterochromatin that is formed dynamically in response to environmental signals by a process that does not require RNAi, and is enriched in histone H3 methylated on lysine 9 (H3K9me).

All heterochromatin is H3K9 dependent.
Not all  heterochromatin is formation is RNAi independent.

So, based on the def of "Heterochromatin island"  heterochromatin island assembly does not seem different from "heterochromatin assembly".

However, I suspect there is something more about "Heterochromatin island" 
If it is required we need a better differentia for heterochromatin island.

@mah11  curated these - is there anything else that will differentiate?

**Comment by ValWood** [comment]
While we decide  if we need to do anything with 

heterochromatin island assembly (GO:1902794)
while this term is under  review can we place it under 
heterochromatin assembly.
It is not there currently.

**Comment by mah11** [comment]
I don't know much beyond I put in earlier comments on this ticket ([here](https://github.com/geneontology/go-ontology/issues/11247#issuecomment-124916159), [here](https://github.com/geneontology/go-ontology/issues/11247#issuecomment-124916162), and [here](https://github.com/geneontology/go-ontology/issues/11247#issuecomment-124916164). 

I used PMIDs 23151475 and 24210919 for the terms in question. I'm neutral on whether to keep distinct CC terms (without them, ~10 FYPO terms lose logical defs; no big deal), but the assembly processes do seem to differ -- the PMID:24210919 intro says 

"S. pombe contains three distinct types of heterochromatin, which differ in their dependency on trans-acting factors."

and then goes on to provide some details.

**Comment by ValWood** [comment]
@pgaudet an we keep for now.? Once we have purged the silencing stuff, I will ask the heterochromatin community to review so that we have everything classified correctly  by the different processes (mechanisms). I think this (mechanism) is really more important than the location, although sometimes we are using one as a proxy for the other.

Possibly these domains are more related to  constitutive heterochromatin assembly.
Once the other changes are in place it will be easier to decide about this. Maybe we will rename  the term, I m not sure.

This is how our annotation looks now ( I fixed a few more today so it will be better tomorrow).
https://www.pombase.org/term/GO:0031507

This also needs the remaining subtelomeric heterochromatin terms and the heterochromatin island term under this branch. 
Once this is done I can ask for community input.

I can also look how the gene products in the individual processes intersect so I can look for any inconsistencies.

At that point I can open new tickets. So I would move this under heterochromatin assembly and close this ticket .

**Comment by ValWood** [comment]
Oh we still need to deal with the fact that  "heterochromatin assembly" and "heterochromaitn island assembly"  are defined identically

The problem is that "heterochromatin island" does not have sufficent differentia from heterochromatin. 
Heterochromatin also fits this definition. @mah any suggestion  how to  differentiate?


This def
A region of heterochromatin that is formed dynamically in response to environmental signals by a process that does not require RNAi, and is enriched in histone H3 methylated on lysine 9 (H3K9me).

would also fit subtelomeric  heterochromatin which is also dependent on environmental signals, and is  RNAi independent.

What else does it need? Maybe  something could be added such as
"regions of heterochromatin interspersed in euchromatic regions" ?

**Comment by mah11** [comment]
> @mah any suggestion how to differentiate?

I just mentioned in [my previous comment](https://github.com/geneontology/go-ontology/issues/11247#issuecomment-621727894) that relevant info can be found in PMID:24210919 and the papers cited in its introduction (and maybe there'll be more in any papers that cite it in turn; I haven't looked into that).

**Comment by ValWood** [comment]
so PMID:24210919  says


The first type corresponds to major H3K9me peaks at centromeres, telomeres, and the mating-type locus. RNAi proteins Argonaute (Ago1) and Dicer (Dcr1) and RNA-dependent RNA polymerase (Rdp1) target transcripts produced by dg/dh repeats in these regions to generate siRNAs that facilitate loading of the Clr4/Suv39h methyltransferase (Reyes-Turcu and Grewal, 2012). The second type includes small blocks of heterochromatin islands, which include meiotic genes and other loci (Cam et al., 2005, Zofall et al., 2012). Islands are akin to facultative heterochromatin and are dynamically regulated in response to environmental signals (Zofall et al., 2012). The assembly of heterochromatin islands at meiotic genes requires Red1 and Rrp6, but not RNAi (Hiriart et al., 2012, Tashiro et al., 2013, Zofall et al., 2012). The third type of heterochromatin consists of domains called HOODs (heterochromatin domains), which are detected under specific growth conditions and are dynamically regulated (Yamanaka et al., 2013). HOODs preferentially assemble at sexual differentiation genes and retrotransposons and require RNAi as well as elimination factors (Yamanaka et al., 2013). Red1 and its partners Pla1 and Pab2 are cofactors for both RNAi and the exosome, which act in parallel to degrade transcripts from loci within these domains (Yamanaka et al., 2013).

Which implies subtelomeric heterochromatin is also RNAi dependent.


islands form at meiotic genes in mitotic cells (facultative)


Suggest
This def
A region of *facultative RNAi-independent*  heterochromatin that is formed dynamically *at specific loci *in response to environmental signals.

Then the  "heterochromatin island assembly" term will also make sense.

**Comment by pgaudet** [comment]
@ValWood also asked to move 
subtelomeric heterochromatin assembly
and
heterochromatun island assembly
under
heterochromatin assembly

**Comment by pgaudet** [comment]
Fixed defintion of 'heterochromatin island' as follows: 'A region of facultative heterochromatin formed dynamically at specific loci in response to environmental signals, independently of RNAi.'

**Comment by ValWood** [comment]
also
heterochromatin domain assembly
under heterochromatin assembly!

**Comment by pgaudet** [comment]
> heterochromatin domain assembly
under heterochromatin assembly!

Yes, done.
